website/client/store/actions/user.js

```diff
@@ -46,3 +46,13 @@ export async function deleteWebhook (store, payload) {
   let response = await axios.delete(`/api/v3/user/webhook/${payload.webhook.id}`);
   return response.data.data;
 }
+
+
+export async function togglePinnedItemAsync (store, params) {
+  let response = await axios.get(`/api/v3/user/toggle-pinned-item/${params.key}`);
```

Comment:
can we also remove the pinned item immediately without waiting for the server response? so we don't have to wait for a roundtrip to the server before updating the UI